full-service/src/service/account.rs

```diff
@@ -1225,6 +1205,38 @@ mod tests {
         )
         .unwrap();
 
-        assert_eq!(unspent_txos.len(), 2);
+        assert_eq!(unspent_txos.len(), 1);
+
+        service
+            .assign_address_for_account(&account_id, None)
+            .unwrap();
+
+        let unverified_txos = Txo::list_unverified(
+            Some(&account_id.to_string()),
+            None,
+            None,
+            None,
+            None,
+            None,
+            None,
+            &mut wallet_db.get_pooled_conn().unwrap().deref_mut(),
+        )
+        .unwrap();
+
+        assert_eq!(unverified_txos.len(), 1);
+
+        let txo_synced_2 = TxoSynced {
+            tx_out_public_key: RistrettoPublic::try_from(&orphaned_txos[0].public_key().unwrap())
+                .unwrap()
+                .into(),
+            key_image: KeyImage::from(rng.next_u64()),
+        };
+
+        service
+            .sync_account(&account_id, vec![txo_synced_2])
```

Comment:
What in this test actually verifies what sync_account does?

Reply:
Have updated to test that unverified gets marked as unspent, as expected